### PR TITLE
[Edit] Kotlinx:datetime

### PR DESCRIPTION
**Pull request by caupolicandiaz**
### Description

This PR makes edits to a pair of entries in the datetime library. The imports in the examples have been updated to use the correct library, one of the examples was revised, and some additional changes were made to address the syntax descriptions and other components of the entry. 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [/ ] All writings are my own.
- [/ ] My entry follows the Codecademy Docs style guide.
- [/ ] My changes generate no new warnings.
- [/ ] I have performed a self-review of my own writing and code.
- [/ ] I have checked my entry and corrected any misspellings.
- [/ ] I have made corresponding changes to the documentation if needed.
- [/ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [/ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [/ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by dakshdeepHERE — APPROVED** [review]
Looks good for 2nd review.

File: content/kotlin/concepts/datetime/terms/daysUntil/daysUntil.md [hunk 1]
```diff
@@ -17,36 +17,31 @@ The **`.daysUntil()`** method calculates the number of whole days between two in
...
+import kotlinx.datetime.*
+
 fun main() {
-    val t1 = Instant.parse("2008-09-29T12:00:00Z")
-    val t2 = t1.plus(Duration.ofHours(125))
-    val t3 = t2.daysUntil(t1)
+    val t1 = Instant.parse("2024-01-01T12:00:00Z")
+    val t2 = Instant.parse("2023-11-01T12:00:00Z")
+    val defaultTZ = TimeZone.currentSystemDefault()
+    val t3 = t2.daysUntil(t1, defaultTZ)
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/datetime/terms/daysUntil/daysUntil.md:38]: Everything is fine but can we use better variable names? Like startTime endTime and t3 is basically diff btw t1 n t2 so variable name related to that

**Review by SSwiniarski — APPROVED** [review]
@caupolicandiaz Looks good to me.

File: content/kotlin/concepts/datetime/terms/daysUntil/daysUntil.md (hunk 1 above)
Comment (SSwiniarski) [reply content/kotlin/concepts/datetime/terms/daysUntil/daysUntil.md:38]: Deferring merge so @caupolicandiaz can address this.

Comment (dakshdeepHERE) [reply content/kotlin/concepts/datetime/terms/daysUntil/daysUntil.md:38]: Yep

**Pull request merged**